lib/manageiq/session.rb

```diff
@@ -0,0 +1,48 @@
+require "extensions/session_extension"
+
+module ManageIQ
+  module Session
+    FakeRequest = Struct.new(:env)
+
+    def self.store
+      @store ||= fetch_store_from_rails
+    end
+
+    def self.store=(session_store)
+      adapter_klass = "ManageIQ::Session::#{session_store.to_s.camelize}Adapter".safe_constantize
+      raise ArgumentError, "invalid session store adapter: #{session_store.inspect}" unless adapter_klass
+
+      configure_session_store(adapter_klass.new)
+
+      @store = nil
+    end
+
+    def self.configure_session_store(adapter)
+      Vmdb::Application.config.session_store(*adapter.session_store_config_args)
+      msg = "Using session_store: #{Vmdb::Application.config.session_store}"
+      _log.info("MIQ(SessionStore) #{msg}")
+      puts "** #{msg}" if !Rails.env.production? && adapter.type != :mem_cache_store
+    end
+
+    def self.revoke(session_ids)
+      store.delete_sessions(session_ids)
+    end
```

Comment:
Super minor, but if we change this to the following we can support single id, multiple id, and Array of id on the interface.
```suggestion
    def self.revoke(*session_ids)
      store.delete_sessions(session_ids.flatten)
    end
```

Or you could push the splat down even to the adapters.

Reply:
Yeah, we can do that.  I was considering it in my head, but was trying to keep the interface simple, but since I got confirmation on this being a good idea from you, I will do this.

That said, was thinking of using `Array.wrap`, but I guess `.flatten` does make more sense.

Reply:
Done.